### PR TITLE
movie detail

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/74040)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image
image
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

**Comment by namnn17b3** [comment]
ready

File: src/views/movie-detail.ejs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  <!-- nav bar -->
+  <%- include('./header') %>
+  <!-- end of navbar -->
+  <div id="movie-id" class="d-none"><%- movie.id %></div>
+
+  <div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true" style="margin: 0 auto;">
+    <div class="modal-dialog modal-lg" style="max-width: 70vw; height: 85vh;">
+      <div class="modal-content" style="width: 100%; height: 100%;">
+        <div class="modal-header d-flex">
+          <h5 class="modal-title mx-auto" id="exampleModalLabel" style="font-size: 25px;"><%- movie.name %></h5>
```
Comment (huongnt-2545) [inline src/views/movie-detail.ejs]: move css vào file internal

File: src/views/movie-detail.ejs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          <button type="button" class="close ml-0 close-video-modal" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <div id="player"></div>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-success btn-cancle btn-back invisible" data-dismiss="modal">BACK</button>
```
Comment (huongnt-2545) [inline src/views/movie-detail.ejs]: I18n

File: src/services/movie.service.ts [hunk 3]
```diff
@@ -62,4 +69,79 @@ export class MovieService {
...
+        .getQuery();
+
+      queryBuilder
+        .addSelect(`subquery${item}.star${item}`, `movie_star${item}`)
+        .leftJoin(
+          `(${subquery})`,
+          `subquery${item}`,
+          `subquery${item}.id = movie.id`,
+        );
+    });
```
Comment (huongnt-2545) [inline src/services/movie.service.ts:96]: tách private method. return querybuilder

File: src/services/review.service.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      queryBuilder
+        .orderBy('review.star', 'DESC')
+        .addOrderBy('review.updatedAt', 'DESC');
+    }
+
+    if (criteriaDate) {
+      queryBuilder
+        .orderBy('review.updatedAt', 'DESC')
+        .addOrderBy('review.star', 'DESC');
+    }
```
Comment (huongnt-2545) [inline src/services/review.service.ts:47]: tách private method

**Review by huongnt-2545 — COMMENTED** [review]
Pull có rất nhiều file changes, pull này c tạo điều kiện review, những pull sau nếu thay đổi > 20 file changes thì em phải tách nhỏ pull ra e nhé

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**